clusters/variables.tf

```diff
@@ -9,14 +9,15 @@ variable "cluster_count" {
   description = "Number of clusters to provision"
 }
 
-variable "ec2_instances_per_cluster" {
-  type        = number
-  description = "Number of EC2 instances per cluster"
```

Comment:
Why is this and `k3s_per_node` being removed?

Reply:
These vars aren't actually used in the `clusters` module.